src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -95,5 +90,19 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
                        "should be emitted to the console and log file (if appropriate). " +
                        "Valid values: ScanTime.")]
         public virtual IEnumerable<string> Traces { get; set; }
+
+        [Option(
+            "level",
+            Separator = ';',
+            Default = new FailureLevel[] { FailureLevel.Error, FailureLevel.Warning },
+            HelpText = "Filter results. Valid values: None, Note, Warning, and Error.")]
```

Comment:
>Filter results. Valid values: None, Note, Warning, and Error. [](start = 24, length = 61)

"Filter output of scan results to one or more failure levels. Valid values: Error, Warning and Note."

Reply:
Done.

---
In reply to: [559750572](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r559750572) [](ancestors = 559750572)